src/controllers/itemDetails/index.js

```diff
@@ -1718,15 +1718,29 @@ function renderCollectionItemType(page, parentItem, type, items) {
     imageLoader.lazyChildren(collectionItems.lastChild);
 }
 
-function renderMusicVideos(page, item, user) {
-    ServerConnections.getApiClient(item.ServerId).getItems(user.Id, {
-        SortBy: 'SortName',
-        SortOrder: 'Ascending',
-        IncludeItemTypes: 'MusicVideo',
-        Recursive: true,
-        Fields: 'PrimaryImageAspectRatio,BasicSyncInfo,CanDelete,MediaSourceCount',
-        AlbumIds: item.Id
-    }).then(function (result) {
+function renderMusicVideos(page, item, user, isAlbum) {
+    let request = {};
+    if (isAlbum) {
+        request = {
+            SortBy: 'SortName',
+            SortOrder: 'Ascending',
+            IncludeItemTypes: 'MusicVideo',
+            Recursive: true,
+            Fields: 'PrimaryImageAspectRatio,BasicSyncInfo,CanDelete,MediaSourceCount',
+            AlbumIds: item.Id
+        };
```

Comment:
Most of this is shared between the two branches here. I would suggest moving everything common to the original assignment and just doing `request.AlbumIds = item.Id` here for example. That should also let you make the `request` variable a `const` since you aren't reassigning it.